numpy/lib/npyio.py

```diff
@@ -506,6 +506,20 @@ def save(file, arr, allow_pickle=True, fix_imports=True):
     Notes
     -----
     For a description of the ``.npy`` format, see :py:mod:`numpy.lib.format`.
+        
+    Any data saved to the file is appended to the end of the file. 
+    Example:
+    --------
+    >>> with open('test.npy', 'wb') as f:
+    ...     np.save(f, np.array([1, 2]))
+    ...     np.save(f, np.array([1, 3]))    
+    
+    >>> with open('test.npy', 'rb') as f:
+    ...     a = np.load(f)
+    ...     b = np.load(f)
+    >>> print(a, b)
+    # [1 2] [1 3]
+    
```

Comment:
This example should be merged with the `Examples` section below, only one examples section is allowed. I also think the doctest will fail, since the `print(a, b)` will expect the exact output of `print(a, b)`. You should try it before pushing to github with `python3 runtests.py --refguide-check`